app/sprinkles/account/src/Repository/PasswordResetRepository.php

```diff
@@ -33,7 +33,9 @@ class PasswordResetRepository extends TokenRepository
     protected function updateUser(UserInterface $user, $args)
     {
         $user->password = Password::hash($args['password']);
+        $user->flag_password_reset_required = false;
         // TODO: generate user activity? or do this in controller?
```

Comment:
It doesn't look like it is happening in controller or here, so I would say no.

Reply:
It would make sense to log some type of message "User set a new password".

Then the controller would be responsible for logging activity messages related to why a user needs to reset their password.